LamestWebserver/LamestWebserver/HttpResponse.cs

```diff
@@ -111,8 +119,19 @@ public byte[] GetPackage()
 
             if(Range != null)
             {
+                if(Range.Item2 >= _contentLength) 
```

Comment:
Range.item is a index an contentLength is a size :D index 100 is not in a contentlength of 100 (index 0-99)